plugins/workloads/go.mod

```diff
@@ -1,12 +1,11 @@
-module github.com/fairwindsops/insights-plugins/workload
+module github.com/fairwindsops/insights-plugins/workloads
```

Comment:
you probably have to change folder name to `workload` here

Reply:
I think it's fine - we're not using this as a go library anywhere. LMK if you think otherwise